### PR TITLE
Placement swipe view

### PR DESCRIPTION
**Pull request by frederik0312**
:) Vertical Swipe

### CODE REVIEW

File: BDSA2020.Entities/Context.cs [hunk 1]
```diff
@@ -29,6 +29,7 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
             // Boilerplade code taken from the Lecture04/Program.cs 
             var configuration = new ConfigurationBuilder().AddUserSecrets(typeof(Context).Assembly).Build();
             var connectionString = configuration.GetConnectionString("ConnectionString");
+           // var connectionString = @"Server=(localdb)\MSSQLLocalDB;Database=BDSA2020;Trusted_Connection=True;";
```
Comment (simoboye) [inline BDSA2020.Entities/Context.cs:32]: Remove

File: BDSA2020.View/Pages/PlacementDescriptionView.razor [hunk 2]
```diff
@@ -8,6 +128,42 @@
...
+                    <td>@student.Id</td>
+                    <td>@student.Degree</td>
+                    <td>@student.MaxWorkingHours</td>
+                </tr>
+            }
+        </tbody>
+    </table>
+    */
+}
+
```
Comment (simoboye) [inline BDSA2020.View/Pages/PlacementDescriptionView.razor:156]: Deleted outcommented code

File: BDSA2020.View/Shared/NavMenu.razor [hunk 3]
```diff
@@ -33,6 +33,11 @@
         </li>
         <li class="nav-item px-3">
             <NavLink class="nav-link" href="placementdescription">
+                <span class="oi oi-book" aria-hidden="true"></span> Placements Description
```
Comment (simoboye) [inline BDSA2020.View/Shared/NavMenu.razor:36]:
```suggestion
                <span class="oi oi-book" aria-hidden="true"></span> Placement Descriptions
```
Is is not placement descriptions? :)

File: BDSA2020.View/wwwroot/css/placementStyle.css [hunk 4]
```diff
@@ -1,15 +1,26 @@
+/*new shit*/
```
Comment (simoboye) [inline BDSA2020.View/wwwroot/css/placementStyle.css:1]: Probably best to not have this comment, although it is correct ;)

File: BDSA2020.View/Pages/PlacementDescriptionView.razor [hunk 5]
```diff
@@ -51,6 +207,7 @@ else
 
     protected override async Task OnInitializedAsync()
     {
+         students = await httpClient.GetFromJsonAsync<Student[]>("StudentRepository");
```
Comment (simoboye) [inline BDSA2020.View/Pages/PlacementDescriptionView.razor:210]: To much indentation here

File: BDSA2020.View/Pages/PlacementDescriptionView.razor [hunk 6]
```diff
@@ -1,4 +1,124 @@
...
+                    </ul>
+                    <div class="divider"></div>
+                    <div style="height: 20px;"></div>
+                    <div class="btnPlacement more">
+                        <p class="center">More</p>
+                    </div>
+                <p class="boldText">Last apply date: 12th Nov 2020</p>
+            </div>
+        </div>
+</div>
```
Comment (simoboye) [inline BDSA2020.View/Pages/PlacementDescriptionView.razor:102]: You can fetch the data from the API. The data is already fetched in the `placementDescriptions` field.
If this is to much we can make a new task in the kanban board for this.

**Review by simoboye — APPROVED** [review]
Good work.
Please see my small comments below :)

**Review by bjornarjatten — COMMENTED** [review]
With bugfixes and comments this should work

**Review by bjornarjatten — APPROVED** [review]
Approved

**Pull request merged**